lib/name_drop/client.rb

```diff
@@ -37,9 +38,9 @@ def delete(endpoint)
 
     def request(method, endpoint, attributes = {})
       response = RestClient::Request.execute(create_request_hash(method, endpoint, attributes))
-      JSON.parse(response) unless response.empty?
+      JSON.parse(response).with_indifferent_access unless response.empty?
     rescue RestClient::ExceptionWithResponse => err
-      raise NameDrop::Error.new(error_message(method), JSON.parse(err.response))
+      raise NameDrop::Error.new(error_message(method), JSON.parse(err.response).with_indifferent_access)
```

Comment:
@adambedford We could with a`.try(:with_indifferent_access)` otherwise we're effectively typecasting what can be sent to `NameDrop::Error.detail`